ext/settings.html

```diff
@@ -3213,6 +3221,87 @@ <h1>Yomitan Settings</h1>
     </div>
 </div></div>
 
+<!-- Generate anki deck modal -->
+<div id="generate-anki-deck-modal" class="modal" tabindex="-1" role="dialog" hidden><div class="modal-content modal-content-full">
+    <div class="modal-header"><div class="modal-title">Anki Deck Generator</div></div>
+    <div class="modal-body generate-anki-deck-layout">
+        <div class="generate-anki-deck-info">
+            <p class="warning-text">
+                WARNING: This feature is experimental!
+            </p>
+            <p>
+                Enter a newline separated list of terms below to send directly to an Anki deck or export to an Anki deck file in <code>Notes in plain text (.txt)</code> format.
+            </p>
+        </div>
+        <textarea autocomplete="off" spellcheck="false" id="generate-anki-deck-textarea" class="no-wrap margin-above" data-tab-action="indent,4"></textarea>
+        <div class="generate-anki-deck-test-container margin-above">
+            <p>
+                Active Anki deck: <code id="generate-anki-deck-active-deck"></code><br>
+                Active Anki card format: <code id="generate-anki-deck-active-card-format"></code>
+            </p>
+            <div class="generate-anki-deck-test-table margin-above">
+                <div class="generate-anki-deck-test-table-header">Test word</div>
+                <div></div>
+                <input type="text" id="generate-anki-deck-test-text-input" class="form-control" value="読め" placeholder="Preview text" autocomplete="off" lang="ja">
+                <button type="button" id="generate-anki-deck-test-render-button">Test</button>
+            </div>
+        </div>
+        <div class="code margin-above" id="generate-anki-deck-render-result"><em>Card render result</em></div>
+    </div>
+    <div class="modal-footer">
+        <button type="button" class="low-emphasis" id="generate-anki-deck-send-to-anki-button">Send to Anki</button>
+        <button type="button" class="low-emphasis" id="generate-anki-deck-export-button">Export to File</button>
+        <button type="button" data-modal-action="hide">Close</button>
+    </div>
+</div></div>
+
+<div id="generate-anki-deck-send-to-anki-modal" class="modal" tabindex="-1" role="dialog" hidden><div class="modal-content modal-content-small">
+    <div class="modal-header"><div class="modal-title">Send Words to Anki</div></div>
+    <div class="modal-body">
+        <p>
+            Are you sure you want to send <strong id="generate-anki-deck-send-wordcount"></strong> words to <code id="generate-anki-deck-active-deck-confirm"></code>? This action cannot be undone.
+        </p>
+        <div class="settings-item margin-above"><div class="settings-item-inner">
+            <div class="settings-item-left">
+                <div class="settings-item-label">
```

Comment:
Why is this setting in the confirmation modal and not in the original modal?

Reply:
It only applies to the `Send to Anki` option and not the `Export to File` option. I don't want to confuse users thinking they should be getting media when exporting to file.